dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.org.eclipse.microprofile.rest.client-1.0.feature

```diff
@@ -3,7 +3,7 @@ symbolicName=com.ibm.websphere.appserver.org.eclipse.microprofile.rest.client-1.
 singleton=true
 -features=com.ibm.websphere.appserver.javax.cdi-1.2; ibm.tolerates:=2.0, \
  com.ibm.websphere.appserver.javax.annotation-1.2; ibm.tolerates:=1.3, \
- com.ibm.websphere.appserver.jaxrsClient-2.0; ibm.tolerates:=2.1, \
+ com.ibm.websphere.appserver.javax.jaxrs-2.0; ibm.tolerates:=2.1, \
  com.ibm.websphere.appserver.jsonp-1.0; ibm.tolerates:=1.1
```

Comment:
We are still pulling in the JSON-P API+Impl here.  It's not a big deal since JSON-P is pretty lightweight.